src/sqlfluff/rules/L012.py

```diff
@@ -9,6 +9,24 @@ class Rule_L012(Rule_L011):
     NB: This rule inherits its functionality from obj:`Rule_L011` but is
     separate so that they can be enabled and disabled separately.
 
+    | **Anti-pattern**
+    | In this example, the alias 'alias_col' is implicit.
```

Comment:
As 'alias_col' is not mentioned (do you mean the code is called that), let's just remove that from the anti-pattern example:

```suggestion
    | In this example, the alias for column 'a' is implicit.
```

Reply:
Good catch